synapse/lib/stormtypes.py

```diff
@@ -8453,7 +8454,7 @@ async def set(self, name, valu):
         else:
             self.runt.user.confirm(('trigger', 'set', name), gateiden=viewiden)
 
-        await self.runt.snap.view.setTriggerInfo(trigiden, name, valu)
+        await view.setTriggerInfo(trigiden, name, valu)
```

Comment:
Can this potentially cause a desync issue with mirrors having triggers successfully update while failing to update on the leader? Seems like we might need the migration to be a nexus event.

Reply:
Actually I suppose the wrong view would be in the nexus event so it should fail on the mirror as well, nevemind.